src/applications/claims-status/containers/ClaimEstimationPage.jsx

```diff
@@ -13,21 +12,18 @@ class ClaimEstimationPage extends React.Component {
     setUpPage();
   }
   render() {
-    const claimType = !this.props.loading ? getClaimType(this.props.claim) : '';
     return (
       <div>
         <div className="row">
           <div className="medium-12 columns">
-            <Breadcrumbs>
-              <li>
-                <Link to="your-claims">Your Claims</Link>
-              </li>
-              <li>
-                <Link to={`your-claims/${this.props.params.id}/status`}>
-                  Your {claimType} Claim
-                </Link>
-              </li>
-            </Breadcrumbs>
+            <ClaimsBreadcrumbs>
+              <Link to={`your-claims/${this.props.params.id}/status`}>
+                Status Details
```

Comment:
Was it intentional to change the breadcrumb text for this page? The old description seems better to me, but I don't know the rationale for the change.

Reply:
@jbalboni Yes, this was a conscious choice, based on several conversations with @mnorthuis and @bethpotts regarding link text.

Reply:
@jbalboni Yes, this was a conscious choice, based on several conversations with @mnorthuis and @bethpotts regarding link text.